.gitignore

```
@@ -171,6 +172,9 @@ port/omrsyslogmessages.rc
 port/MSG00001.bin
 x64/
 
+# CLion
+.idea
```

To be consistent with other ignored directories, I think the this should be:
```suggestion
.idea/
```